pahfit/base.py

```diff
@@ -564,3 +569,69 @@ def read(self, filename, tformat=None):
         readout = (bb_info, df_info, h2_info, ion_info, att_info)
 
         return readout
+
+    def starting_point(self, obs_x, obs_y, param_info):
```

Comment:
The codacy "error" is that this function does not use self, so it does not need to be in this class.  It is recommending a stand alone function as a result.  Otherwise, you could declare this function as a static.  See https://realpython.com/instance-class-and-static-methods-demystified/

Reply:
I've changed it to a staticmethod.